### ISSUE TITLE
fan_modes and swing_modes not working

### ISSUE THREAD

**Issue by firstcolle** [jcwillox/hass-template-climate#75]
### The problem

I can't get fan_modes and swing_modes working.

i set the template like this:
`  modes:
    - "auto"
    - "heat"
    - "cool"
    - "dry"
    - "fan_only"
    - "off"
  fan_modes:
    - "auto"
    - "low"
    - "high"
  swing_modes:
    - "horizontal"
    - "vertical"
 `
but i see only the havac mode in the card:
![image](https://github.com/user-attachments/assets/fc989be0-ccbd-43b9-b430-0de73bf8a805)

### What type of installation are you running?

None

**Comment by Tamas-Toth-ebola** [comment]
Did you set the corresponding value templates also?

**Comment by firstcolle** [comment]
Yes, but I solved my issues with two cards. One card is a climate template just for sensors and temperature settings, and the other one is for climate control. Then, using card-mod, I edited the height of each card to display only what I need.
